R/RLearner_FDAclassif_knn.R

```diff
@@ -0,0 +1,45 @@
+#' @title Learner for knn on functional data.
+#'
+#' @description
+#' Learner for knn on functional data.
+#'
+#' @export
+makeRLearner.fdaclassif.knn = function() {
+  makeRLearnerClassif(
+    cl = "fdaclassif.knn",
+    package = "fda.usc",
+    par.set = makeParamSet(
+      makeIntegerLearnerParam(id = "knn", lower = 1L, default = NULL, special.vals = list(NULL)),
+      makeDiscreteLearnerParam(id = "metric", default = "metric.lp", values = list("metric.lp", "metric.kl",
```

Comment:
use vector `c("metric.lp", "metric.kl", "metric.hausdorff", "metric.dist")` instead of list. Check this everywhere.